src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Looks like this got included accidentally. Can you remove it?

Reply:
sure, i'll get to it ASAP